### PR TITLE
INFRA-837: Add sonar integration

### PR DESCRIPTION
**Pull request by jluque0101**
This PR enables sonar integration.

### CODE REVIEW

**Comment by jluque0101** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/sonar.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Sonar
+on:
+  push:
+    branches:
+      - master
```
Comment (sethkfman) [inline .github/workflows/sonar.yml:5]: This should be `main`

Comment (jluque0101) [reply .github/workflows/sonar.yml]: Thanks @sethkfman , is updated.

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**